### PR TITLE
Add Lingo.app v1.0

### PR DESCRIPTION
**Pull request by danielbayley**
Lingo app from the Noun Project.

Signed-off-by: Daniel Bayley daniel.bayley@me.com

### CODE REVIEW

File: Casks/lingo.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
+cask 'lingo' do
+  version :latest
+  sha256 :no_check
+
+  url 'https://nounproject.s3.amazonaws.com/lingo/Lingo.dmg'
```
Comment (vitorgalvao) [inline Casks/lingo.rb]: Needs a `# nounproject.s3.amazonaws.com/lingo was verified as official when first introduced to the cask` comment above this line ([example](https://github.com/caskroom/homebrew-cask/blob/4ad02c9f087deb9e553ab0129c079d73c474add8/Casks/0ad.rb#L5); [reasoning](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/url.md#when-url-and-homepage-hostnames-differ-add-a-comment)).

File: Casks/lingo.rb [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  name 'Lingo'
+  homepage 'https://lingoapp.com'
+  license :gratis
+
+  app 'Lingo.app'
+
+  zap delete: [
+                '~/Library/Preferences/com.lingoapp.Lingo.plist',
+                '~/Library/Application Support/com.lingoapp.Lingo',
+                '~/Library/Application Support/com.apple.sharedfilelist/com.apple.LSSharedFileList.ApplicationRecentDocuments/com.lingoapp.lingo.sfl'
```
Comment (vitorgalvao) [inline Casks/lingo.rb]: Needs a comma at the end. To solve common stylistic issues, run `brew cask style --fix {{your_cask_file}}` prior to submission.

**Comment by danielbayley** [comment]
@vitorgalvao Should be sorted now :ok_hand:

**Comment by vitorgalvao** [comment]
Thank you for the contribution. It was merged directly as c56ba9db113a7453475a35ba70541e7ba580201c to keep commit history cleaner. Your contribution is still credited to you.

**Pull request closed without merging**